src/test/java/sk/stuba/fei/uim/vsa/pr1/ThesisTest.java

```diff
@@ -508,7 +508,7 @@ void TH09_shouldUpdateThesis() {
             String teacherField = teacherFields.get(0);
             assertNotNull(getFieldValue(thesis, teacherField, teacherClass));
             assertEquals(teacher01Id, getEntityId(getFieldValue(thesis, teacherField, teacherClass), teacherIdField));
-            thesis = setFieldValue(thesis, teacherField, teacher02);
+            thesis = setFieldValue(thesis, teacherField, teacher02);  // zakazal som kradnutie prace a teda zmenu ucitela
```

Comment:
Nebral by som to ako kradnutie práce. Zmena vedúceho práce je bežná záležitosť, ktorá má svoje opodstatnenie. Nakoľko učiteľ je iba jeden z atribútov práce, ak zachováme, že môže byť v práci uvedený práve jeden je to absolútne v poriadku.

Reply:
Ja som to tak žiaľ bral, tak teda som to mal asi tušiť ospravedlňujem sa za zbytočný pull request.